playbooks/technology_radar/artsy-tech-radar.csv

```diff
@@ -59,4 +59,5 @@ Kafka,hold,tools,FALSE,
 MongoDB,hold,tools,FALSE,
 Pingdom,hold,tools,TRUE,
 Semaphore,hold,tools,FALSE,
-Travis CI,hold,tools,TRUE,
\ No newline at end of file
+Travis CI,hold,tools,TRUE,
+Heroku,hold,platforms,FALSE
```

Comment:
A short explanation could help here. Borrowing from the kubernetes line above, something like: "We prefer kubernetes where we have shared solutions for authorization, logging, monitoring, alerting, scaling, and routing."

Reply:
```suggestion
Heroku,hold,platforms,FALSE,"We prefer kubernetes where we have shared solutions for authorization, logging, monitoring, alerting, scaling, and routing."
```